app/src/main/res/drawable/shape.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:padding="@dimen/dimen_10dp"
+    android:shape="rectangle">
+    <!-- you can use any color you want I used here gray color-->
+    <solid android:color="#ABABAB" />
+    <corners android:radius="@dimen/dimen_12dp" />
+</shape>
```

Comment:
ეს რისი შეიფია?

Reply:
done in this pr: https://github.com/iTarley/QuizApp-SpaceIntern/pull/6